mvc/models/model-binding.rst

```diff
@@ -1,8 +1,113 @@
-.. include:: /../common/stub-topic.txt
+Model Binding
+=============
 
-|stub-icon| Model Binding Request Data
-======================================
+By `Rachel Appel <http://github.com/rachelappel>`_
 
-.. include:: /../common/stub-notice.txt
+In this article
 
-.. _issue: https://github.com/aspnet/Docs/issues/114
+- `Introduction to model binding`_
+- `How model binding works`_
+- `Customize default model binding behavior with attributes`_
+- `Binding formatted data from the request body`_ 
+- `Resources`_
+
+Introduction to model binding
+-----------------------------
+Model binders are objects in the ASP.NET MVC framework that map data from HTTP requests to action method parameters. The parameters may be simple types such as strings, integers, or floats, or they may be complex objects representing an enterprise-sized model. This is a great feature of MVC because mapping incoming data to a counterpart is an often repeated scenario, regardless of size or complexity of the data. MVC solves this problem by abstracting binding away so developers don't have to keep rewriting a slightly different version of that same code in every app. In previous versions of ASP, including Classic ASP and WebForms, you would extract the HTTP POST or GET values manually, validate the data, and then send that data off to the database. Not much of the process was automated, so you would often see code similar the following example that captures two form values, puts them in variables and TextBox objects, storing them for their journey to the database:
+
+.. literalinclude:: model-binding/sample/src/MVCMovie/retro-sample.txt
+   :language: html
+
+The above code quickly becomes repetitive, cumbersome, and error prone to write, even when it's just a few fields. Real world apps often have many fields combined with complex structures. Admittedly, this kind of mapping code is a bit tedious to write as well. 
```

Comment:
> Admittedly, ...

If we stick w/ a comparison like this, there's no reason to admit anything. Tedious is an obvious fallout from "repetitive, cumbersome, and error-prone". (Hmm, is my hyphenated error-prone an error?)